### DISCUSSION TITLE
WebUI

### DISCUSSION THREAD

**Discussion by ralf2201** [gravitational/teleport#8525] [Q&A]
Hi, in my WebUi of teleport the button Add Server is greyed out. Can someone tell me what i have to do to bring it to work.

![image](https://user-images.githubusercontent.com/66718931/136541148-36695060-d1b0-47e8-ae05-618a7861ca7e.png)

**Comment by dudeisbrendan03** [comment]
If you're using Teleport OSS you will have to add the server with tctl, unless you're an enterprise subscriber or use Teleport Cloud, a lot of the UI features are indeed disabled or unavailable.
